### DISCUSSION TITLE
Mark subscriber as 'exclude forever' (extra option in  addition to active / inactive

### DISCUSSION THREAD

**Discussion by uildriks** [hizzle-co/noptin#655] [Ideas]
I deleted  ericjonesmyemail@gmail.com, but next day he was in the subscribers again. I want such 'subscribers' to exclude once and for all. Keeping him on 'inactive' he remains in view each time I review my list of new subscribers.

**Comment by picocodes** [comment]
We'll work on such a feature. For now, you can create an automation rule that deletes a new subscriber if their email address matches ericjonesmyemail@gmail.com

**Comment by picocodes** [comment]
We're adding a "Blocked" status starting with Noptin version 3.8.6:- https://noptin.com/guide/email-subscribers/blocking-subscribers/
